### ISSUE TITLE
Cannot reproduce paper results

### ISSUE THREAD

**Issue by theophilegervet** [guhur/hiveformer#6]
Hi Pierre-Louis, 

I'm a big fan of your work, loved the Airbert idea as well!

I'm trying to reproduce the results of your work, specifically Table 2.
I'm training tasks independently using the default parameters in `train.py` except the learning rate, which I've set to `5e-5` as in the paper.

My results so far:
push_button: 100%
reach_target: 100%
every other task: somewhere between 0% and 50%, very far from the paper results

@guhur Are we supposed to be able to reproduce the paper results with the code you made available?
Am I missing something?

**Comment by theophilegervet** [comment]
![Screen Shot 2023-01-24 at 9 33 51 PM](https://user-images.githubusercontent.com/14276221/214467810-5e3a0d33-4ac5-41fd-a045-56b4f8b0767f.png)

**Comment by guhur** [comment]
Following our private discussion, you told me you were able to reproduce results using my original un-cleaned code.
Unfortunately, I will be unable to find time digging in the difference, so I have rather pushed my original code here: https://github.com/guhur/hiveformer/tree
